message CarRepeat {

  repeated CarSeat seats = 1;
  optional string name = 2;

}

message CarSeat {
  required string color = 1;
}